aptos-move/block-executor/src/proptest_types/types.rs

```diff
@@ -130,11 +166,24 @@ impl<K: Hash + Clone + Debug + Eq + PartialOrd + Ord> ModulePath for KeyType<K>
     }
 }
 
-#[derive(Debug, Clone, PartialEq, Eq)]
+#[derive(Derivative)]
+#[derivative(Debug(bound = ""), PartialEq(bound = ""), Eq(bound = ""))]
```

Comment:
What are these bounds?

Reply:
apparently we no longer need PartialEq and Eq for ValueType, so this just disappears.
otherwise, it'd be better to implement them manually (as opposed to skipping the field that doesn't implement it by default).